### PR TITLE
[DATA-567] Support running cartographer through rdk

### PR DESCRIPTION
**Pull request by tessavitabile**
Together with https://github.com/viamrobotics/rdk/pull/1507, this lets us run cartographer through rdk.

### CODE REVIEW

**Review by JohnN193 — APPROVED** [review]

File: slam-libraries/viam-cartographer/src/slam_service/config.cc [hunk 1]
```diff
@@ -36,9 +40,6 @@ int ParseAndValidateConfigParams(int argc, char** argv) {
     } else if (FLAGS_port.empty()) {
         LOG(ERROR) << "-port is missing.\n";
         return EXIT_FAILURE;
-    } else if (FLAGS_sensors.empty()) {
-        LOG(ERROR) << "-sensors is missing.\n";
-        // return EXIT_FAILURE;
     }
```
Comment (kkufieta) [inline slam-libraries/viam-cartographer/src/slam_service/config.cc:43]: [q] Why did you remove this?

**Review by kkufieta — COMMENTED** [review]
Nice, just one question

File: slam-libraries/viam-cartographer/src/slam_service/config.cc (hunk 1 above)
Comment (tessavitabile) [reply slam-libraries/viam-cartographer/src/slam_service/config.cc:43]: This error was shown if sensors was empty, which is expected for offline.

**Review by kkufieta — APPROVED** [review]
LGTM
